cpp/include/raft/random/rmat_rectangular_generator.cuh

```diff
@@ -84,6 +139,38 @@ void rmat_rectangular_gen(IdxT* out,
  * distributions across all the scales
  */
 template <typename IdxT, typename ProbT>
+void rmat_rectangular_gen(const raft::handle_t& handle,
+                          raft::random::RngState& r,
+                          raft::device_vector_view<IdxT, IdxT> out,
+                          raft::device_vector_view<IdxT, IdxT> out_src,
+                          raft::device_vector_view<IdxT, IdxT> out_dst,
+                          ProbT a,
+                          ProbT b,
+                          ProbT c,
+                          IdxT r_scale,
+                          IdxT c_scale,
+                          IdxT n_edges)
+{
```

Comment:
Same here- are we able to at least do validation of the edge list sizes? And can the edge list sizes replace the explicit n_edges argument?

Reply:
I've added size validation and removed the explicit `n_edges` parameter of both overloads; thanks!